Binance.Net/Objects/Futures/FuturesData/BinanceFuturesTrade.cs

```diff
@@ -55,7 +55,7 @@ public class BinanceFuturesTrade
         /// <summary>
         /// Realized pnl
         /// </summary>
-        public decimal RealizedPnl { get; set; }
+        public decimal RealizedPnL { get; set; }
```

Comment:
I've renamed all PnL properties to Pnl so its consistent. Leave this at Pnl please

Reply:
ok, i'll change it